docs/sources/configure/_index.md

```diff
@@ -3946,6 +3950,25 @@ embedded_cache:
   # CLI flag: -<prefix>.embedded-cache.ttl
   [ttl: <duration> | default = 1h]
 
+lru_cache:
+  # Maximum memory size of the whole cache. A unit suffix (KB, MB, GB) may be
+  # applied. Default: 1GB.
+  # CLI flag: -<prefix>.lrucache.max-size-bytes
+  [max_size_bytes: <int> | default = 0B]
```

Comment:
The defaults at line 3955 and line 3957 still don't match.  =(

Same issue with line 3960 and 3962.

Is there some other file that needs to be edited to update the defaults?

Reply:
I found a way of fixing it, sorry for the delay :smile: